src/toil/provisioners/aws/awsProvisioner.py

```diff
@@ -78,13 +88,41 @@ def getNodeShape(self, preemptable=False):
                      disk=(instanceType.disks * instanceType.disk_capacity * 2 ** 30))
 
     @classmethod
-    def sshLeader(cls, clusterName):
+    def _buildContext(cls, clusterName, zone=None):
+        if not zone:
+            zone = os.environ.get('TOIL_AWS_ZONE', None)
+        if not zone:
+            zone = boto.config.get('Boto', 'ec2_region_name')
+            if zone is not None:
+                zone += 'a'  # derive an availability zone in the region
```

Comment:
This case is only if the user specifies a region in their boto config and nothing else. In that case I'm defaulting their zone to the 'A' zone in the specified region.